src/rpc/client.http.ts

```diff
@@ -4,31 +4,30 @@ import { Effect, Layer, LogLevel, Logger, ManagedRuntime } from "effect";
 import { type CallRemoteFn, type ClientContext, catchPromise } from "./client.common.js";
 import { Procedures } from "./procedure.js";
 
-const ClientLayer = Layer.empty.pipe(
-  Layer.merge(FetchHttpClient.layer),
-  Layer.provide(Logger.minimumLogLevel(LogLevel.Trace)),
-  Layer.provide(Logger.pretty),
-);
-
-const ClientRuntime = ManagedRuntime.make(ClientLayer);
-
 class Client extends Effect.Service<Client>()("Client", {
   scoped: Effect.gen(function* () {
-    const rpc = yield* RpcClient.make(Procedures).pipe(
-      Effect.provide(RpcClient.layerProtocolHttp({ url: "/api/rpc" })),
```

Comment:
Effect.provide should be avoided here, as it will shutdown the layer once the effect finishes (which is immediately here)